### PR TITLE
fix(docs): add content to usage-scenario page

### PR DESCRIPTION
**Pull request by dhmlau**
Currently the [Usage scenario page](https://loopback.io/doc/en/lb4/Usage-scenarios.html) is empty. This PR adds some content in the page which serves as an index besides the sidebar. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Usage-scenarios.md [hunk 1]
```diff
@@ -5,3 +5,13 @@ keywords: LoopBack 4.0, LoopBack-Next
...
 ---
+
+There are various usage scenario of using LoopBack, including accessing
+databases, integrating to other infrastructures and calling other services.
+
+- [create other forms of APIs](Create-other-forms-of-apis.md)
+- [access databases](Access-databases.md)
+- [call other services](Calling-other-APIs-and-Web-Services.md)
+- [integrate with infrastructures](Integrate-with-infrastructures.md)
+- [deployment](Deployment.md)
```
Comment (raymondfeng) [inline docs/site/Usage-scenarios.md]: I'm not sure if `deployment` belongs here. If yes, it should be the last item and the title should be more specific such as `deploy to cloud`

Comment (dhmlau) [reply docs/site/Usage-scenarios.md]: +1 on `deploy to cloud`.  
`Deployment` is under `usage scenario` in the sidebar.  If it doesn't belong there, maybe we can move it somewhere.

File: docs/site/Usage-scenarios.md [hunk 2]
```diff
@@ -5,3 +5,13 @@ keywords: LoopBack 4.0, LoopBack-Next
...
 ---
+
+There are various usage scenario of using LoopBack, including accessing
+databases, integrating to other infrastructures and calling other services.
+
+- [create other forms of APIs](Create-other-forms-of-apis.md)
+- [access databases](Access-databases.md)
+- [call other services](Calling-other-APIs-and-Web-Services.md)
+- [integrate with infrastructures](Integrate-with-infrastructures.md)
+- [serving static files](Serving-static-files.md)
```
Comment (raymondfeng) [inline docs/site/Usage-scenarios.md]: let's use `serve static files`

File: docs/site/Usage-scenarios.md [hunk 3]
```diff
@@ -5,3 +5,13 @@ keywords: LoopBack 4.0, LoopBack-Next
...
+- [deployment to cloud](Deployment.md)
```
Comment (raymondfeng) [inline docs/site/Usage-scenarios.md]: To be consistent with other items, let's use `deploy to cloud`

**Pull request merged**